src/envo/envo-edit.owl

```diff
@@ -44006,6 +44014,55 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105226> (soil layer bottom depth)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105226> "A depth which inheres in the bottom of a soil layer."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105226> <https://orcid.org/0000-0001-6284-4821>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105226> "2021-12-19T21:19:05Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105226> "soil layer bottom depth"@en)
+EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_06105226> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_06105223> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/PATO_0001595>)))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105266> (soil profile)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://www.nrcs.usda.gov/wps/portal/nrcs/detail/soils/edu/?cid=nrcs142p2_054308") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105266> "The soil profile is a vertical section of the soil and made up of layers, running parallel to the surface, called soil horizons or soil layers."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105266> <https://orcid.org/0000-0001-6284-4821>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105266> "2021-12-23T04:06:18Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105266> "soil profile"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105266> <http://purl.obolibrary.org/obo/ENVO_00001998>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105267> (soil profile characterization)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://www.ctahr.hawaii.edu/mauisoil/a_profile.aspx") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105267> "A planned process in which the soil profile is characterized, describing the different soil horizons that composed it."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105267> <https://orcid.org/0000-0001-6284-4821>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105267> "2021-12-23T04:10:36Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105267> "soil profile characterization"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105267> <http://purl.obolibrary.org/obo/OBI_0000011>)
```

Comment:
I am not enthused about adding more investigative processes to ENVO, there is no clear dividing line between ENVO and OBI here. But we can add contingent we may need to offload these later

Reply:
We need that term has we need to record when the process happens. But maybe you can suggest a better way for handling time here.

Reply:
>I am not enthused about adding more investigative processes to ENVO, there is no clear dividing line between ENVO and OBI here.

It's not a biomedical investigation, so it's fine in ENVO. If that branch grows enough, we would spin off a new ontology for environmental assessments and investigations.